cmd/scan.go

```diff
@@ -1102,6 +1108,17 @@ func isScanReleaseFailed(release *client.ReleaseStatus) error {
 		failedScans["IAC"] = release.IAC.ScanID
 	}
 
+	if breakByScannerType {
+		scannerType := strings.ToUpper(scan.ScannerType)
+
+		if _, ok := failedScans[scannerType]; !ok {
+			// This means, this scanner type isn't failed. So, we can ignore it because we are breaking by scanner type
+			return nil
+		} else {
+			return errors.New(fmt.Sprintf("project does not pass release criteria due to [%s] failure", scannerType))
```

Comment:
you can use fmt.Errorf("")

Reply:
%101 right

Reply:
https://github.com/kondukto-io/kdt/pull/89/commits/f2aa322c1234eb3a523e902edce92eacc782c554